flake/dev/all-maintainers.nix

```
@@ -0,0 +1,57 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      apps.all-maintainers = {
+        program = pkgs.writeShellApplication {
+          name = "update-all-maintainers";
+          runtimeInputs = with pkgs; [
+            git
+            coreutils
+          ];
+          text = ''
+            root="$(git rev-parse --show-toplevel)"
+            install -m 644 -T \
+              ${config.packages.all-maintainers} \
+              "$root/stylix/generated/all-maintainers.nix"
+          '';
+        };
+        meta.description = "update all-maintainers.nix";
+      };
+
+      packages.all-maintainers =
+        let
+          meta = pkgs.callPackage ../../stylix/meta.nix { };
+        in
+        pkgs.runCommand "all-maintainers"
+          {
+            passAsFile = [ "maintainers" ];
+            maintainers = lib.pipe meta [
+              builtins.attrValues
+              (builtins.concatMap (value: value.maintainers or [ ]))
+              (map (value: {
+                name = value.github;
+                inherit value;
+              }))
+              builtins.listToAttrs
+              (lib.generators.toPretty { })
+            ];
+            inherit (config.treefmt) projectRootFile;
+            nativeBuildInputs = [ config.formatter ];
+          }
+          ''
+            touch "$projectRootFile"
+
+            {
+              echo '# THIS FILE IS GENERATED with \`nix run .#all-maintainers\`'
```

No need to escape backticks within single quotes:

```suggestion
              echo '# THIS FILE IS GENERATED with `nix run .#all-maintainers`'
```